src/Books.css

```diff
@@ -7,3 +7,20 @@
 body {
   background-color: rgb(248, 253, 253);
 }
+
+/* categories loading
+@keyframes spin {
+  0% { transform: rotate(0deg); }
+  100% { transform: rotate(360deg); }
+}
+
+.spinner {
+  border: 4px solid rgba(0, 0, 0, 0.1);
+  border-left: 4px solid #3498db;
+  border-radius: 50%;
+  width: 30px;
+  height: 30px;
+  animation: spin 1s linear infinite;
+  margin: 0 auto;
+}
+*/
```

Comment:
- [ ] For best practices and clean code, kindly ensure that you remove all the commented and unused code lines from your code base.